tests/upgrades/test_containers_removal.py

```diff
@@ -121,30 +123,35 @@ def test_pre_scenario_containers_support_removal(self):
                 dockerhub_container.compute_resource.id, compute_resource.id)
 
             # container from external registry
-            external_container = entities.DockerRegistryContainer(
-                command='sleep inf',
-                compute_resource=compute_resource,
-                organization=[org],
-                registry=registry,
-                repository_name=repo_name,
-            ).create()
-            self.assertEqual(
-                external_container.compute_resource.id, compute_resource.id)
-            self.assertEqual(external_container.registry.id, registry.id)
-            self.assertEqual(external_container.repository_name, repo_name)
+            if rh_registry_available:
+                external_container = entities.DockerRegistryContainer(
+                    command='sleep inf',
+                    compute_resource=compute_resource,
+                    organization=[org],
+                    registry=registry,
+                    repository_name=repo_name,
+                ).create()
+                self.assertEqual(
+                    external_container.compute_resource.id, compute_resource.id)
+                self.assertEqual(external_container.registry.id, registry.id)
+                self.assertEqual(external_container.repository_name, repo_name)
 
             running_containers = docker_host.run('docker ps')
             self.assertEqual(running_containers.return_code, 0)
 
             self.assertTrue(any(dockerhub_container.name in line
                                 for line in running_containers.stdout))
-            self.assertTrue(any(external_container.name in line
-                                for line in running_containers.stdout))
+            if rh_registry_available:
+                self.assertTrue(any(external_container.name in line
+                                    for line in running_containers.stdout))
+
+            ext_container_name = external_container.name if rh_registry_available else ''
 
             scenario_dict = {self.__class__.__name__: {
+                'rh_registry_available': rh_registry_available,
                 'docker_host': docker_host.hostname,
                 'dockerhub_container': dockerhub_container.name,
-                'external_container': external_container.name,
+                'external_container': ext_container_name,
```

Comment:
Once the bug is verified, this will be '', Are you ok for that ?

Reply:
This bug should be fixed in z-stream, so it will probably only take a week or two for it to move from VERIFIED to CLOSED. If we have some info it will take longer than that, I am very much OK with another PR that would revert this patch.

But my guess/worry is that we will release 6.6 before this bug is CLOSED. And this test is going to be removed after 6.6 GA anyway.